### DISCUSSION TITLE
The relationships between the tables

### DISCUSSION THREAD

**Discussion by Fatima-Alzahraa-da** [Budibase/budibase#9068] [Help]
I'm a beginner at budibase. I'm having problems creating relationships between the tables... I want to create an application to track expenses and income in general about managing a budget... I've designed a table for expenses, income, and methods of payment... The relationship between income and payment method, how will it be? Am I creating a related field from the expense table? Also, the expense related to the user Do I create a relationship field in the expense table? Or vice versa
![Screenshot 2022-12-15 220413](https://user-images.githubusercontent.com/55523991/208048157-8d694096-6abe-48f6-8f85-ae6e082529b7.png)
![Screenshot 2022-12-16 104457](https://user-images.githubusercontent.com/55523991/208048774-a670e0a4-fd25-4bf4-ba9f-9bf74819765c.png)

**Comment by andz-bb** [comment]
hey @Fatima-Alzahraa-da 

in your case those Create Column dialogs look about right for the sort of relationships you want to create. When you create a column like that it will also display a column in the related table. So in your first example, there would be a column created in the `EXPENSES` table called `PAYMENT_EXPENSES` (this could also just be called `PAYMENT`) and then another column called `EXPENSES` would display in the `PAYMENT` table. In each case the display members of the linked rows will show in the relationship columns.

Hope this clears things up a bit for you. We have full docs on these relationships here if you want to have a look: https://docs.budibase.com/docs/relationships
